plugin-server/src/worker/ingestion/event-pipeline/3-pluginsProcessEventStep.ts

```diff
@@ -13,7 +15,7 @@ export async function pluginsProcessEventStep(
     let processedEvent: PluginEvent | null = event
 
     // run processEvent on all events that are not $snapshot
-    if (event.event !== '$snapshot') {
+    if (!EVENTS_TO_IGNORE.includes(event.event)) {
```

Comment:
Aside, is there a test to check that we are indeed not running plugins processing on these events?

Reply:
I don't think there is but there probably should be!